### PR TITLE
[RVPS] Integrate in AS-gRPC, RVPS binary and documents

### PR DESCRIPTION
**Pull request by Xynnn007**
This PR include RVPS-related things:
- Add RVPS binary
- Let AS support two ways to interact with RVPS, s.t. integrate inside and by gRPC. This can be controlled by rust feature
- Document about RVPS

### CODE REVIEW

**Comment by Xynnn007** [comment]
The error is caused by the upstream change (virtee-kbs types), I'll open another pr to fix this.

File: Cargo.toml [hunk 1]
```diff
@@ -6,10 +6,19 @@ edition = "2021"
...
+[[bin]]
+name = "rvps"
+path = "bin/rvps/src/main.rs"
+required-features = [ "rvps-server", "rvps-proxy", "tokio/rt-multi-thread" ]
 
 [features]
-default = [ "in-toto" ]
-in-toto = []
+default = [ "rvps-server" ]
+rvps-server = [ "in-toto" ]
```
Comment (jialez0) [inline Cargo.toml:18]: Why not directly use `"dep:in-toto` here, so that `in-toto` feature can be remolved below.

Comment (Xynnn007) [reply Cargo.toml:18]: My initial idea is to make `in-toto` as a option feature. Here I made a mistake, let me fix into
```
default = [ "rvps-server" ]
rvps-server = []
rvps-proxy = [ "tonic" ]

in-toto = [ "dep:in-toto" ]
```

In this way, we can choose whether to enable `in-toto` support

**Comment by jialez0** [comment]
@Xynnn007 Thanks! Could you please update the architecture diagram in README.md to follow this PR?

**Review by jialez0 — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**